### PR TITLE
Fix issue DPTP-3664 Add ci-scheduling-webhook to build09

### PR DESCRIPTION
**Pull request by bear-redhat**

### CODE REVIEW

File: clusters/build-clusters/build09/ci-scheduling-webhook/dns.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# https://github.com/openshift/enhancements/blob/master/enhancements/dns/configurable-dns-pod-placement.md#proposal
+# dns-default-* pods do not tolerate all taints. This means our infra nodes and nodes tainted to support
+# pod classes are not getting these pods scheduled onto them.
+# This seemed to lead to https://bugzilla.redhat.com/show_bug.cgi?id=2086887 .
+apiVersion: operator.openshift.io/v1
+kind: DNS
```
[the remark this answers is not in the document]
Comment (bear-redhat) [reply clusters/build-clusters/build09/ci-scheduling-webhook/dns.yaml:6]: This config exists in the original `ci-scheduling-webhook` (https://github.com/jupierce/ci-tools/blob/ci-scheduling-webhook/cmd/ci-scheduling-webhook/res/dns.yaml) repo and applied to all build farms. I am not sure whats the impact for not applying it.

[the remark this answers is not in the document]
Comment (hongkailiu) [reply clusters/build-clusters/build09/ci-scheduling-webhook/dns.yaml:6]: > applied to all build farms

Shouldnt it then be in some common folder instead?

> I am not sure whats the impact for not applying it.

Could you please ask Justin about it?

[the remark this answers is not in the document]
Comment (bear-redhat) [reply clusters/build-clusters/build09/ci-scheduling-webhook/dns.yaml:6]: It was, until b05 is using a different config.

[the remark this answers is not in the document]
Comment (hongkailiu) [reply clusters/build-clusters/build09/ci-scheduling-webhook/dns.yaml:6]: Ack.

https://github.com/openshift/release/blob/master/clusters/build-clusters/build05/ci-scheduling-webhook/dns.yaml.md

What is the real thing for b05?

[the remark this answers is not in the document]
Comment (hongkailiu) [reply clusters/build-clusters/build09/ci-scheduling-webhook/dns.yaml:6]: One more:
If it is the same on every one except b05, let symlink it.

**Pull request closed without merging**